apps/fulltextsearch.sh

```diff
@@ -87,51 +89,167 @@ fi
 
 # Check & install docker
 install_docker
+install_docker_compose
 set_max_count
-mkdir -p "$RORDIR"
-docker pull "$nc_fts"
-
-# Create configuration YML
-cat << YML_CREATE > /opt/es/readonlyrest.yml
```

Comment:
we should probably remove that file on old installations if they reinstall with this script?

Reply:
solved by 74616ea